library/core/src/num/nonzero.rs

```diff
@@ -287,6 +287,130 @@ nonzero_integers_div! {
     NonZeroUsize(usize);
 }
 
+macro_rules! nonzero_integers_add {
+    ( $( $Ty: ident($Int: ty); )+ ) => {
+        $(
+
+            impl $Ty {
+                /// Add a positive integer to a non-zero value.
+                /// Return None on overflow.
```

Comment:
Don't hesitate to add intra doc links everywhere, it's always nice to have more links to relevant docs 😀

```suggestion
                /// Return [`None`] on overflow.
```

Reply:
Okay, great, I was unaware of those. I'm adding this for all `None`s in the doc comments :)